gen-phpweb-sqlite-db.php

```diff
@@ -164,6 +164,31 @@ function scan($dir, $lang)
 			$dbh->exec("INSERT INTO fs (lang, prefix, keyword, name, prio) values ('$lang', '$prefix', '" . metaphone($keyword) . "', '$doc_rel', ".($prio+10).")");
 
 		}
+
+		if ($f === 'reserved.keywords.php') {
+			// Yes, this is fragile.
+			$handle = fopen($file, 'r');
```

Comment:
There is no error handling here.

Reply:
What sort of handling would you like to see here? Would it be enough to move this into the above `if` statement?

Reply:
At least not try to run any code if you can't open the file. Right now ,that would error on line 173 with the `fgets`.